GitSearch/Networking/Service/SearchServices.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  PostService.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/26/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+struct SearchServices{
+    
+    static let shared = SearchServices()
+    
+    let searchSession = URLSession(configuration: .default)
+    
+    func getRepositories(keysSequence: String, _ completion: @escaping (Result<SearchInfo?, HTTPNetworkError>) -> ()) {
```

Comment:
It looks like misusage when we have optional in result (`Result<SearchInfo?, HTTPNetworkError>`). The purpose of [`Result`](https://developer.apple.com/documentation/swift/result) class is to provide _either_ result value or error. If you don't have proper result, you should not provide nil, but error instead.

Reply:
done